### PR TITLE
check thread and reaction from channel config

### PR DESCRIPTION
**Pull request by adrian09h**
# Submit a pull request

- **Check threadable and reactionable from channel config**

### CODE REVIEW

File: library/src/main/java/com/getstream/sdk/chat/view/Dialog/MoreActionDialog.java [hunk 1]
```diff
@@ -170,8 +170,15 @@ private boolean copyableMessage() {
 
     private boolean threadableMessage() {
         return style.isEnableThread()
+                && viewModel.getChannel().getConfig().isReplies()
```
Comment (tbarbugli) [inline library/src/main/java/com/getstream/sdk/chat/view/Dialog/MoreActionDialog.java]: `isReplies` is a pretty terrible name; what about `repliesEnabled` instead

File: library/src/main/java/com/getstream/sdk/chat/view/Dialog/MoreActionDialog.java [hunk 2]
```diff
@@ -170,8 +170,15 @@ private boolean copyableMessage() {
...
                 && !viewModel.isThread();
     }
+
+    private boolean reactionableMessage() {
```
Comment (tbarbugli) [inline library/src/main/java/com/getstream/sdk/chat/view/Dialog/MoreActionDialog.java]: naming `canReactOnMessage` or similar sounds better

File: library/src/main/java/com/getstream/sdk/chat/view/Dialog/MoreActionDialog.java [hunk 3]
```diff
@@ -170,8 +170,15 @@ private boolean copyableMessage() {
...
+        return style.isEnableReaction()
```
Comment (tbarbugli) [inline library/src/main/java/com/getstream/sdk/chat/view/Dialog/MoreActionDialog.java]: `isEnableReaction` -> `isReactionEnabled`

File: library/src/main/java/com/getstream/sdk/chat/view/Dialog/MoreActionDialog.java (hunk 1 above)
Comment (adrian09h) [reply library/src/main/java/com/getstream/sdk/chat/view/Dialog/MoreActionDialog.java]: `isReplies` was created by _Generate_ _Getter_ and _Setter_ automatically.
Sure we can change it as `repliesEnabled`

**Pull request merged**